templates/taxi/driver_detail.html

```diff
@@ -2,12 +2,23 @@
 
 {% block content %}
   <h1>
-    Username: {{ driver.username }}
+    Username: {{ driver.username }} 
+    <a href="{% url 'taxi:driver-delete' driver.id %}" class="btn btn-danger">
```

Comment:
Ensure that the URL tag uses single quotes consistently for better readability. Change `{% url "taxi:driver-delete" driver.id %}` to `{% url 'taxi:driver-delete' driver.id %}`.